pkg/search/query.go

```diff
@@ -25,49 +25,50 @@ type Query struct {
 }
 
 type Qualifiers struct {
-	Archived         *bool
-	Assignee         string
-	Author           string
-	Base             string
-	Closed           string
-	Commenter        string
-	Comments         string
-	Created          string
-	Draft            *bool
-	Followers        string
-	Fork             string
-	Forks            string
-	GoodFirstIssues  string
-	Head             string
-	HelpWantedIssues string
-	In               []string
-	Interactions     string
-	Involves         string
-	Is               []string
-	Label            []string
-	Language         string
-	License          []string
-	Mentions         string
-	Merged           string
-	Milestone        string
-	No               []string
-	Org              string
-	Project          string
-	Pushed           string
-	Reactions        string
-	Repo             []string
-	Review           string
-	ReviewRequested  string
-	ReviewedBy       string
-	Size             string
-	Stars            string
-	State            string
-	Status           string
-	Team             string
-	Topic            []string
-	Topics           string
-	Type             string
-	Updated          string
+	Archived            *bool
+	Assignee            string
+	Author              string
+	Base                string
+	Closed              string
+	Commenter           string
+	Comments            string
+	Created             string
+	Draft               *bool
+	Followers           string
+	Fork                string
+	Forks               string
+	GoodFirstIssues     string
+	Head                string
+	HelpWantedIssues    string
+	In                  []string
+	Interactions        string
+	Involves            string
+	Is                  []string
+	Label               []string
+	Language            string
+	License             []string
+	Mentions            string
+	Merged              string
+	Milestone           string
+	No                  []string
+	Org                 string
+	Project             string
+	Pushed              string
+	Reactions           string
+	Repo                []string
+	Review              string
+	ReviewRequested     string
+	TeamReviewRequested string
```

Comment:
Nit: would be nice to keep this list properly sorted

Reply:
Thanks for your feedback it has been changed